### PR TITLE
adds cassandra-medusa and its dep packages

### PR DESCRIPTION
**Pull request by mritunjaysharma394**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

**Comment by mritunjaysharma394** [comment]
With the latest commit, have reduced the packages in this PR with the ones which weren't building to separately handle them

File: py3-cassandra-medusa.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Generated from https://pypi.org/project/cassandra-medusa/
+package:
+  name: py3-cassandra-medusa
+  version: 0.17.1
+  epoch: 0
+  description: Apache Cassandra backup and restore tool
+  copyright:
+    - license: Apache
```
Comment (mamccorm) [inline py3-cassandra-medusa.yaml:8]: Looks like the license name was not created correctly here, can we check it and complete?

File: py3-azure-core.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Generated from https://pypi.org/project/azure-core/
+package:
+  name: py3-azure-core
+  version: 1.29.6
+  epoch: 0
+  description: Microsoft Azure Core Library for Python
+  copyright:
+    - license: MIT License
```
Comment (mamccorm) [inline py3-azure-core.yaml:8]: Think this should just be 'MIT'

File: py3-azure-identity.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# Generated from https://pypi.org/project/azure-identity/
+package:
+  name: py3-azure-identity
+  version: 1.15.0
+  epoch: 0
+  description: Microsoft Azure Identity Library for Python
+  copyright:
+    - license: MIT License
```
Comment (mamccorm) [inline py3-azure-identity.yaml:8]: This should just be 'MIT', which is the spdx short identifier:
https://spdx.org/licenses/MIT.html

File: py3-cassandra-medusa.yaml (hunk 1 above)
Comment (mritunjaysharma394) [reply py3-cassandra-medusa.yaml:8]: Yes, its Apache License 2.0, I will update this, thanks Mark

File: py3-azure-storage-blob.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
+# Generated from https://pypi.org/project/azure-storage-blob/
+package:
+  name: py3-azure-storage-blob
+  version: 12.19.0
+  epoch: 0
+  description: Microsoft Azure Blob Storage Client Library for Python
+  copyright:
+    - license: MIT License
```
Comment (mamccorm) [inline py3-azure-storage-blob.yaml:8]: This should just be 'MIT', which is the spdx short identifier:
https://spdx.org/licenses/MIT.html

File: py3-azure-core.yaml (hunk 2 above)
Comment (mritunjaysharma394) [reply py3-azure-core.yaml:8]: Cool, will fix it, thanks for noticing it

File: py3-azure-storage-blob.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  - runs: |
+      cd sdk/storage/azure-storage-blob
+      python setup.py build
+      python setup.py install --prefix=/usr --root="${{targets.destdir}}"
+
+  - uses: strip
+
+update:
+  enabled: true
+  manual: false
```
Comment (mamccorm) [inline py3-azure-storage-blob.yaml:45]: QQ - what does having enabled: true and manual:false do here?

I'm asking as i'm not so sure - vs a suggestion to change

File: py3-boto3.yaml [hunk 6]
```diff
@@ -0,0 +1,38 @@
+# Generated from https://pypi.org/project/boto3/
+package:
+  name: py3-boto3
+  version: 1.34.17
+  epoch: 0
+  description: The AWS SDK for Python
+  copyright:
+    - license: Apache License 2.0
```
Comment (mamccorm) [inline py3-boto3.yaml:8]: This should just be 'Apache-2.0', which is the spdx short identifier:
https://spdx.org/licenses/Apache-2.0.html

File: py3-cassandra-driver.yaml [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+  version: 3.28.0
+  epoch: 0
+  description: DataStax Driver for Apache Cassandra
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - py3-six
+      - py3-geomet
+      - python-3
```
Comment (mamccorm) [inline py3-cassandra-driver.yaml:13]: This uses python-3 but the other files use python3

File: py3-cassandra-driver.yaml [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+      tag: ${{package.version}}
+
+  - name: Python Build
+    uses: python/build-wheel
+
+  - uses: strip
+
+update:
+  enabled: true
+  manual: false
```
Comment (mamccorm) [inline py3-cassandra-driver.yaml:37]: This should just be 'Apache-2.0', which is the spdx short identifier:
https://spdx.org/licenses/Apache-2.0.html

File: py3-grpcio-health-checking.yaml [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+  version: 1.60.0
+  epoch: 0
+  description: Standard Health Checking Service for gRPC
+  copyright:
+    - license: Apache License 2.0
+  dependencies:
+    runtime:
+      - py3-protobuf
+      - py3-grpcio
+      - python-3
```
Comment (mamccorm) [inline py3-grpcio-health-checking.yaml:13]: This uses python-3 but the other files use python3

File: py3-grpcio-tools.yaml [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+  epoch: 0
+  description: Protobuf code generator for gRPC
+  copyright:
+    - license: Apache License 2.0
+  dependencies:
+    runtime:
+      - py3-protobuf
+      - py3-grpcio
+      - py3-setuptools
+      - python-3
```
Comment (mamccorm) [inline py3-grpcio-tools.yaml:14]: This uses python-3 but the other files use python3

File: py3-msal.yaml [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+  description: The Microsoft Authentication Library (MSAL) for Python library enables your app to access the Microsoft Cloud by supporting authentication of users with Microsoft Azure Active Directory accounts (AAD) and Microsoft Accounts (MSA) using industry standard OAuth2 and OpenID Connect.
+  copyright:
+    - license: MIT
+  dependencies:
+    runtime:
+      - py3-requests
+      - py3-pyjwt
+      - py3-cryptography
+      - py3-mock
+      - python-3
```
Comment (mamccorm) [inline py3-msal.yaml:15]: This uses python-3 but the other files use python3

File: py3-gcloud-aio-storage.yaml [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+       --output-fd 3 3>&1 >&2
+
+       python3 -m installer -d "${{targets.destdir}}" \
+       dist/*.whl
+
+  - uses: strip
+
+update:
+  enabled: true
+  manual: false
```
Comment (mamccorm) [inline py3-gcloud-aio-storage.yaml:52]: same as previous - how do these two tags work together? :)

File: py3-msal.yaml [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+      tag: ${{package.version}}
+
+  - name: Python Build
+    uses: python/build-wheel
+
+  - uses: strip
+
+update:
+  enabled: true
+  manual: false
```
Comment (mamccorm) [inline py3-msal.yaml:39]: same as previous

**Review by mamccorm — APPROVED** [review]

**Pull request merged**